<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
<title>TestNG Report</title>
<style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}.invisible {display:none}</style>
</head>
<body>
<table>
<tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr>
<tr><th colspan="7">Suite</th></tr>
<tr><td><a href="#t0">Create Customer API Test</a></td><td class="num">2</td><td class="num">0</td><td class="num attn">1</td><td class="num">3,192</td><td></td><td></td></tr>
<tr class="stripe"><td><a href="#t1">Delete Customer API Test</a></td><td class="num">0</td><td class="num">0</td><td class="num attn">2</td><td class="num">942</td><td></td><td></td></tr>
<tr><th>Total</th><th class="num">2</th><th class="num">0</th><th class="num attn">3</th><th class="num">4,134</th><th colspan="2"></th></tr>
</table>
<table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">Suite</th></tr></tbody><tbody id="t0"><tr><th colspan="4">Create Customer API Test &#8212; failed</th></tr><tr class="failedeven"><td rowspan="1">com.w2a.APITestingFramework.testcases.CreateCustomerTest</td><td><a href="#m0">validateCreateCustomerAPIWithInValidSecretKey</a></td><td rowspan="1">1625004033878</td><td rowspan="1">1882</td></tr><tr><th colspan="4">Create Customer API Test &#8212; passed</th></tr><tr class="passedeven"><td rowspan="2">com.w2a.APITestingFramework.testcases.CreateCustomerTest</td><td><a href="#m1">validateCreateCustomerAPIWithValidSecretKey</a></td><td rowspan="2">1625004035788</td><td rowspan="2">1250</td></tr><tr class="passedeven"><td><a href="#m2">validateCreateCustomerAPIWithValidSecretKey</a></td></tr></tbody>
<tbody id="t1"><tr><th colspan="4">Delete Customer API Test &#8212; failed</th></tr><tr class="failedeven"><td rowspan="2">com.w2a.APITestingFramework.testcases.DeleteCustomerTest2</td><td><a href="#m3">validateDeleteCustomerAPI</a></td><td rowspan="2">1625004037327</td><td rowspan="2">923</td></tr><tr class="failedeven"><td><a href="#m4">validateDeleteCustomerAPI</a></td></tr></tbody>
</table>
<h2>Create Customer API Test</h2><h3 id="m0">com.w2a.APITestingFramework.testcases.CreateCustomerTest#validateCreateCustomerAPIWithInValidSecretKey</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>{email=idiot1@gmail.com, name=idiot, description=this is me no1}</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [200] but found [401]
	at com.w2a.APITestingFramework.testcases.CreateCustomerTest.validateCreateCustomerAPIWithInValidSecretKey(CreateCustomerTest.java:41)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 15 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m1">com.w2a.APITestingFramework.testcases.CreateCustomerTest#validateCreateCustomerAPIWithValidSecretKey</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>{email=sam1@gmail.com, name=sam, description=this is me no1, address[postal_code]=123.0}</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m2">com.w2a.APITestingFramework.testcases.CreateCustomerTest#validateCreateCustomerAPIWithValidSecretKey</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>{email=sam2@gmail.com, name=samuel, description=this is me no2, address[postal_code]=456.0}</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h2>Delete Customer API Test</h2><h3 id="m3">com.w2a.APITestingFramework.testcases.DeleteCustomerTest2#validateDeleteCustomerAPI</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>{id=cus_Ih4nqWuWxyGD95}</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [200] but found [404]
	at com.w2a.APITestingFramework.testcases.DeleteCustomerTest2.validateDeleteCustomerAPI(DeleteCustomerTest2.java:35)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 15 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m4">com.w2a.APITestingFramework.testcases.DeleteCustomerTest2#validateDeleteCustomerAPI</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>{id=cus_JUcwzQCosxT91x}</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [200] but found [404]
	at com.w2a.APITestingFramework.testcases.DeleteCustomerTest2.validateDeleteCustomerAPI(DeleteCustomerTest2.java:35)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 15 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
</body>
</html>
